java/client/src/org/openqa/selenium/remote/tracing/Tags.java

```diff
@@ -38,6 +50,35 @@ private Tags() {
   };
 
   public static final BiConsumer<Span, HttpResponse> HTTP_RESPONSE = (span, res) -> {
-    span.setAttribute("http.status_code", res.getStatus());
+    int statusCode = res.getStatus();
+    if (res.getTargetHost() != null) {
+      span.setAttribute("http.target_host", res.getTargetHost());
+    }
+    res.getTargetHost();
+    span.setAttribute("http.status_code", statusCode);
+
+    switch (statusCode / 100) {
```

Comment:
This would be better as just status codes rather divided

Reply:
I understand your point. The division is so we can match the error code to a family of error codes rather than a particular error code. I did consider a cleaner way of doing using native java package but we don't have that dependency in the workspace. I think I can leverage existing Netty server response to make the code readable rather that using if-else.